app/components/task_list_items/neighbour_responses_component.rb

```diff
@@ -36,7 +36,9 @@ def status_tag_component
     end
 
     def status
-      if @planning_application.consultation.nil? || @planning_application.consultation.neighbour_responses.none?
+      if @planning_application.consultation.nil? ||
+         @planning_application.consultation.neighbour_responses.none? ||
+         @planning_application.consultation.end_date.nil?
```

Comment:
How about wrapping these conditions on a method under `consultation`?
Maybe it could be `@planning_application.consultation.not_started?`